plenum/common/util.py

```diff
@@ -599,3 +603,9 @@ def check_endpoint_valid(endpoint, required: bool=True):
         raise InvalidEndpointIpAddress(endpoint) from exc
     if not (port.isdigit() and int(port) in range(1, 65536)):
```

Comment:
Can you make a method called valid port which covers these cases, this method can be useful at other places too.